DAS/README.md

```diff
@@ -2,29 +2,122 @@
 
 *This directory contains scripts to log and test various components of the data data-acquisition-system*
```

Comment:
I know you didn't write it, but could you remove the duplicate `data` here?

Reply:
Too easy, thanks!